config.yml

```diff
@@ -1,28 +1,21 @@
 sql:
-    default: 'postgre'
-    postgre:
-        lib: 'psycopg2'
-        args: []
-        kwargs:
-            host: "localhost"
-            port: 5431
-            database: "mimic"
-            user: "mimicuser"
-            password: "mimicuser"
-    fhirbase:
-        lib: 'psycopg2'
-        args: []
-        kwargs:
-            host: "localhost"
-            port: 5433
-            database: "fhirbase"
-            user: "postgres"
-            password: "postgres"
-    oracle:
```

Comment:
I think we want to keep oracle: it's supposed to be an alternative driver to connect to oracle databases when it's not in postgres

Reply:
ok, I removed it since it was not used in the code but it makes sense to keep it indeed.

Reply:
Done.